climmob/tests/test_utils/test_views_api_languages.py

```diff
@@ -0,0 +1,464 @@
+import json
+import unittest
+from unittest.mock import patch, MagicMock
+
+from climmob.views.Api.languages import (
+    readListOfLanguages_view,
+    addLanguageForUse_view,
+    deleteLanguage_view,
+    readListOfUnusedLanguages_view,
+    readAllGeneralPhrases_view,
+    changeGeneralPhrases_view,
+)
+
+
+class TestReadListOfLanguagesView(unittest.TestCase):
+    def setUp(self):
+        self.view = readListOfLanguages_view(MagicMock())
+        self.view.request.method = "GET"
+        self.view.user = MagicMock(login="test_user")
+
+    def mock_translation(self, message):
+        return message
+
+    @patch(
+        "climmob.views.Api.languages.getListOfLanguagesByUser",
+        return_value=[{"lang_code": "en", "lang_name": "English"}],
+    )
+    def test_process_view_success(self, mock_getListOfLanguagesByUser):
```

Comment:
add self.assertTrue(mock_getListOfLanguagesByUser.called), this also fix the IDE warning for non use of parameter. This should be the same in all required cases, if the parameters is not necessary you can replace with "_" or "*_" if you need more than 1 patches